alexagruso/powder-simulator
# Feature/physics

How to test:

Make sure that each physics type works properly (powder, liquid, and gas). The powder is white, the liquid is blue, and the gas is red. The yellow and green elements should act as unbreakable solids.

You should also try to create particle outside of the board and make sure that particles aren't created or that segfaults don't occur.

Please report any errors as well as steps to reproduce the error if possible.
